components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -143,7 +253,17 @@ service Policies {
     option (chef.automate.api.policy).action = "create";
     option (chef.automate.api.iam.policy).resource = "iam:policies:{id}:members";
     option (chef.automate.api.iam.policy).action = "iam:policyMembers:create";
-    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			tags: "policies";
+		};
+  };
+  /*
+  Create a new role
+
+  Creates a new role to be used in the policies that control permissions in Automate.
+
+  A role defines the scope of actions in a policy statement.
```

Comment:
Adapted from the text deleted from the .md file; please add:
"There are several default *Chef-managed* roles that are essential to the operation of Chef Automate and cannot be altered.
Roles that you create are *Custom* roles, and you have full control about modifying or deleting them.